bibliopixel/animation/tests.py

```diff
@@ -5,62 +5,62 @@
 
 class StripChannelTest(BaseStripAnim):
 
-    def __init__(self, led):
-        super().__init__(led)
+    def __init__(self, layout):
+        super().__init__(layout)
         self.internal_delay = 0.500
         self.colors = [colors.Red, colors.Green, colors.Blue, colors.White]
 
     def step(self, amt=1):
 
-        self._led.set(0, colors.Red)
-        self._led.set(1, colors.Green)
-        self._led.set(2, colors.Green)
-        self._led.set(3, colors.Blue)
-        self._led.set(4, colors.Blue)
-        self._led.set(5, colors.Blue)
+        self._layout.set(0, colors.Red)
```

Comment:
As noted above... see? everything uses this. 
In all honesty, why the &%$@ did I make _led 'protected' in the first place. If we are going to change this now, we might as well do right be it so that in BaseAnimation is just self.layout.set